src/Elmish/React/Import.purs

```diff
@@ -25,8 +20,8 @@ type EmptyProps (r :: #Type) = ( | r )
 -- | (including an empty subset) to be passed in.
 type ImportedReactComponentConstructor' reqProps optProps result =
     forall props
-     . IsSubsetOf props (reqProps + optProps + CommonProps)
-    => IsSubsetOf (reqProps ()) props
+     . IsSubsetOf props (reqProps + optProps + ())
+    => IsSubsetOf (reqProps + ()) props
```

Comment:
Curious: Why is `()` needed here? Without understanding this part it looks like `1 + 2 + 0` vs `1 + 2`. Is `()` not the neutral element for `Row`, i.e. empty row?